Design-Docs/entropy.md

```diff
@@ -0,0 +1,75 @@
+---
+title: ADR SCS Entropy
+version: 2023-02-06-001
+authors: Mathias Fechner
+state: v0.9 
+---
+
+## 1. Terminology of Entropy
+
+### 1.1. Entropy in information Technology
+
+    Entrophy is a concecpt that is widely used in a scope of information 
+    technolgy. It is a measure of the amount of disorder or radomness in
+    a system. Entrophy is used to measure the amount of information in a
+    self contained systems, as well as the amount of incertitude that exists
+    in this system. For Cryptographic procedures and operation is a good 
+    entropy a must have !  
+
+    In tradional Baremetal System the amount of incertitude is or was generated by the 
+    radonmes Ready/write cycle of the disk heads of a diskdrive,  bus timeings or
+    Keyboard Timeings.
+
+### 1.2 Entropy in Virtual Instances
+
+     A virtual instance or Virtual machines has all this radomness sources not. A instance will operate normal. But if cryptographic operations will happen,
```

Comment:
A virtual instance or virtual machines does not have this sources for random numbers. An instance will operate normally, but if cryptographic operations will happen[...]